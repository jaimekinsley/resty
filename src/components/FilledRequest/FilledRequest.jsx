import React from 'react';
import PropTypes from 'prop-types';

const FilledRequest = ({ method, api }) => (
  <>
    <p>{method}</p>
    <p>{api}</p>
  </>
);

FilledRequest.propTypes = {
  method: PropTypes.string.isRequired,
  api: PropTypes.string.isRequired
};

export default FilledRequest;
