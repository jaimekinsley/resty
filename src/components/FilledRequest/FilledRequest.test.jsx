import React from 'react';
import { shallow } from 'enzyme';
import FilledRequest from './FilledRequest';

describe('FilledRequest component', () => {
  it('renders a FilledRequest', () => {
    const wrapper = shallow(<FilledRequest method="get" api="https://futuramaapi.com" />);
    expect(wrapper).toMatchSnapshot();
  });
});
