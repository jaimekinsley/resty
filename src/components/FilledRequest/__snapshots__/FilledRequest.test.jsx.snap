// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilledRequest component renders a FilledRequest 1`] = `
<Fragment>
  <p>
    GET
  </p>
  <p>
    https://futuramaapi.com
  </p>
</Fragment>
`;
